### PR TITLE
Update the gateway-certs-generator README.md

### PR DESCRIPTION
**Pull request by uma-kt**
- Rephrase the sentences.
- Update the hyperlinks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,79 +36,82 @@ helm repo update
...
-```console
-git clone https://github.com/kubeslice/gateway-certs-generator.git
-cd gateway-certs-generator
-```
+   ```console
+   git clone https://github.com/kubeslice/gateway-certs-generator.git
+   cd gateway-certs-generator
+   ```
 
 2. Adjust image name variable `IMG` in the [`Makefile`](Makefile) to change the docker tag to be built.
```
Comment (veenamj-avesha) [inline README.md:49]: Modify the image name

File: README.md [hunk 2]
```diff
@@ -36,79 +36,82 @@ helm repo update
...
    Default image is set as `IMG ?= aveshasystems/gateway-certs-generator:latest`. Modify this if required.
```
Comment (veenamj-avesha) [inline README.md:50]: The default image

File: README.md [hunk 3]
```diff
@@ -36,79 +36,82 @@ helm repo update
...
-### Running Local Image on Kind Clusters
+   ```console
+   make docker-build
+   ```
+### Run Local Image on Kind Clusters
 
-1. Loading gateway-certs-generator Image into your kind cluster ([kind](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)).
-   If needed, replace `aveshasystems/gateway-certs-generator` with your locally built image name in the previous step.
+1. Load the gateway-certs-generator image into your kind cluster ([kind](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)).
+   If needed, modify `aveshasystems/gateway-certs-generator` with your locally built image name in the previous step.
```
Comment (veenamj-avesha) [inline README.md:58]: here `replace` is correct

File: README.md [hunk 4]
```diff
@@ -36,79 +36,82 @@ helm repo update
...
+2. Check the loaded image in the cluster. Modify the node name if required.
+
+* Note: `kind-control-plane` is the name of the Docker container. Modify if needed. 
+  
+  ```console
+  docker exec -it kind-control-plane crictl images
+  ```
+
+### Deploy on a Cluster
+1. Create a chart values file `yourvaluesfile.yaml` file. Refer to [values.yaml](https://github.com/kubeslice/charts/blob/master/charts/kubeslice-controller/values.yaml) to update the `kubeslice-controller` image to the local build image.
```
Comment (veenamj-avesha) [inline README.md:79]: chart values file called `yourvalues...` 

delete the file at the end of this sentence. The sentence has 2 instances of `files`.

**Review by veenamj-avesha — COMMENTED** [review]
I have left a few comments.

File: README.md (hunk 3 above)
Comment (veenamj-avesha) [reply README.md:58]: This comment is not fixed yet.

**Review by veenamj-avesha — COMMENTED** [review]
A previous comment is not fixed.

File: README.md [hunk 5]
```diff
@@ -36,79 +36,82 @@ helm repo update
...
+
+2. Check the loaded image in the cluster. Modify the node name if required.
+
+* Note: `kind-control-plane` is the name of the Docker container. Modify if needed. 
+  
+  ```console
+  docker exec -it kind-control-plane crictl images
+  ```
+
+### Deploy Certificate Generator on a Cluster
```
Comment (veenamj-avesha) [inline README.md:78]: Deploy the Certificate Generator

File: README.md (hunk 3 above)
Comment (uma-kt) [reply README.md]: updated.

**Review by veenamj-avesha — APPROVED** [review]
I've left a comment. Please incorporate it before merging this PR. 

Other changes LGTM. Approving this PR.

**Pull request merged**